src/component/contents/DraftEditorBlocks.react.js

```diff
@@ -0,0 +1,210 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
```

Comment:
I am not sure if we need this... if each DraftEditorBlock know that it has children, and if it has a parent, there is no need for such grouping.